app/models/fhir/domain_resource.rb

```diff
@@ -9,9 +9,9 @@ class DomainResource < Resource
 
     def self.transform_json(json_hash, target = DomainResource.new)
       result = self.superclass.transform_json(json_hash, target)
-      result['text'] = Narrative.transform_json(json_hash['text']) unless json_hash['text'].nil?      
+      result['text'] = Narrative.transform_json(json_hash['text']) unless json_hash['text'].nil?
       result['contained'] = json_hash['contained'].map { |var| Resource.transform_json(var) } unless json_hash['contained'].nil?
-      result['extension'] = json_hash['extension'].map { |var| Extension.transform_json(var) } unless json_hash['extension'].nil?
+      # result['extension'] = json_hash['extension'].map { |var| Extension.transform_json(var) } unless json_hash['extension'].nil?
```

Comment:
We need to create separate story/bug for this to investigate. For some reasons this is not working

Reply:
Created [MAT-1337](https://jira.cms.gov/browse/MAT-1337) to investigate and resolve this issue in the generator.